app/src/ui/merge-conflicts/merge-conflicts-dialog.tsx

```diff
@@ -24,8 +24,9 @@ interface IMergeConflictsDialogProps {
   readonly openFileInExternalEditor: (path: string) => void
   readonly externalEditorName?: string
   readonly openRepositoryInShell: (repository: Repository) => void
-  readonly currentBranch: string
-  readonly theirBranch: string
+  readonly ourBranch: string
+  /* `undefined` when we didn't know the branch name upstream */
```

Comment:
Is `upstream` the right phrase here (relating to remotes and tracking branches) or is this related to the context of the merge (being able to convert the `MERGE_HEAD` commit ID to something meaningful to the end user)?

Reply:
ah yeah, poor word choice there